src/metabase/models/revision/diff.clj

```diff
@@ -133,7 +133,12 @@
   The directionality of the statement should indicate that `o1` changed into `o2`."
   [model o1 o2]
   (when-let [[before after] (data/diff o1 o2)]
-    (let [ks         (keys (or after before))
+    (let [all-ks         (keys (or after before))
+          ;; ignore collection_id as part of diff if the dashboard_id has changed
```

Comment:
It's a bit weird right, because when the `dashboard_id` or the `collection_id` changes, we want to generate one message instead of two, but ideally the message might be something like "moved this question from Dashboard D *into Collection C*"? But we'd have to refactor a bit in order to achieve that, because the `collection_id` isn't changing at all... so I think this seems fine for now but maybe we can revisit when we have more time.

Reply:
I would also maybe only do this if `model` is `"Card"`. Just in case we have (or add) some model type with a `collection_id` and `dashboard_id` where both should be reported someday.